__tests__/cpfCnpj/validators/index.test.ts

```diff
@@ -21,3 +21,25 @@ describe('isCnpj', () => {
     expect(isCnpj(param)).toBeFalsy()
   })
 })
+
+describe('isCpf', () => {
+  const validCpf = [
+    '77333151077',
+    '84627346069',
+    '37395050010'
+  ]
+  const invalidCpf = [
+    '77333151078',
+    '479980796',
+    '373.950.500-10',
```

Comment:
is this CPF invalid?

Reply:
Yes, it has a formatting

Reply:
I'm not sure this is a good approach, our function should be smart enough to identify if it's already formatted.